deployment/createS3Bucket.js

```diff
@@ -1,34 +1,33 @@
-const bucketName = process.argv[2];
-
 const aws = require('aws-sdk');
-
 const s3 = new aws.S3();
 
+const bucketName = process.env.AWS_SERVICE_NAME;
+
 s3.listBuckets((err, data) => {
   if (err) {
     console.log('Error', err);
-    return;
+    process.exit(1);
   }
 
   if (data.Buckets.find(bucket => bucket.Name === bucketName)) {
     console.log(`Bucket "${bucketName}" already exists.`);
-  }
-
-  /**
-  Warning: Adding "LocationConstraint" param will break for our region (us-east-1).
-  But, "LocationConstraint" would be required for any other region.
-  https://docs.aws.amazon.com/cli/latest/reference/s3api/create-bucket.html#examples
-  **/
-  const bucketParams = {
-    Bucket: bucketName
-  };
+  } else {
+    /**
+    Warning: Adding "LocationConstraint" param will break for our region (us-east-1).
```

Comment:
Are we doing anything with LocationConstraint?

Reply:
We are not. But it seemed like it could be a potential future "gotcha." This is most probably documentation overkill.